Makefile

```diff
@@ -120,10 +120,10 @@ test-behat:
 	$(EXEC) vendor/bin/behat $(BEHAT_ARGS)
 
 test-phpunit-functional:                                                                               ## Run phpunit fonctional tests
-	$(EXEC) vendor/bin/phpunit --group functional
+	$(EXEC) vendor/bin/phpunit $(PHPUNIT_ARGS) --group functional
 
 tu: vendor app/config/assets_version.yml                                                               ## Run the PHP unit tests
-	$(EXEC) vendor/bin/phpunit --exclude-group functional
+	$(EXEC) vendor/bin/phpunit $(PHPUNIT_ARGS) --exclude-group functional 
```

Comment:
Good idea. Maybe you can create a new step just to run phpunit with argument only and use this new step in `tu` ? WDYT ?